docs/manuals/userman/configuration/config-checker.md

```diff
@@ -24,18 +27,29 @@ Here is the list of the possible issues that the checker could return:
 - Depending on the `storage_type` value of a `DataNodeConfig^`, an `ERROR` issue is created if a specific required
   property is missing.
 - An `ERROR` issue is created if one of the `inputs` and `outputs` parameters of a `TaskConfig^` does not correspond
-  to a `DataNodeConfig`.
+  to a `DataNodeConfig` or a list of `DataNodeConfig^`.
```

Comment:
```suggestion
  to a `DataNodeConfig^`.
```

Reply:
The input and output can be a list right?

Reply:
Oh !

I interpreted "if one of the `inputs` and `outputs` parameters" as "if one of the items of the lists"...

Nevermind then.